pyvista/plotting/renderer.py

```diff
@@ -1259,11 +1281,13 @@ def show_bounds(
             elif location in ('furthest', 'back'):
                 cube_axes_actor.SetFlyModeToFurthestTriad()
             else:
-                raise ValueError(f'Value of location ({location}) not understood.')
+                raise ValueError(
+                    f'Value of location ("{location}") should be either "all", "origin",'
+                    ' "outer", "default", "closest", "front", "furthest", or "back".'
```

Comment:
This list includes `'default'`, `'closest'`, `'furthest'` but these aren't mentioned in the docstring.

Reply:
Fixed in 04db34ed982fe7647212ca4233939c869be538ad.